stock_location_route_description/models/__init__.py

```diff
@@ -0,0 +1 @@
+from . import stock_location_route
```

Comment:
Shouldn't it be stock_location_route_description? The rest LGTM

Reply:
@DavidJForgeFlow in python files you add logic to models, and the model name is stock.location.route, so the python file name should reflect the name of the model you are implementing :)

Reply:
I just saw that the model is named is stock_location_route.py, so that is really my bad :c